apis/scheduling/v1alpha1/types.go

```diff
@@ -62,6 +63,7 @@ type ElasticQuotaStatus struct {
 	Used v1.ResourceList `json:"used,omitempty" protobuf:"bytes,1,rep,name=used,casttype=ResourceList,castkey=ResourceName"`
 }
 
+// +kubebuilder:object:root=true
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
Let's remove k8s tags for kubebuilder owned stuff (CRDs).

Reply:
as I mentioned before, the deepcopy runtime.objects are needed to be created hence we need this annotation. Found it used in the plugins implementations.